From 9db6386ac5ee962861d3913a102c975a7815dd5b Mon Sep 17 00:00:00 2001
From: kiddin9 <48883331+kiddin9@users.noreply.github.com>
Date: Wed, 28 Dec 2022 08:25:36 +0800
Subject: [PATCH] Update Makefile

---
 amule/Makefile | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/amule/Makefile b/amule/Makefile
index b83dcc4539..d90fded6b0 100644
--- a/amule/Makefile
+++ b/amule/Makefile
@@ -31,6 +31,19 @@ define Package/amule
 	+libupnp +libbfd +libcryptopp
 endef
 
+define Download/AmuleWebUI
+  URL:=https://github.com/MatteoRagni/AmuleWebUI-Reloaded/archive
+  URL_FILE:=master.zip
+  FILE:=AmuleWebUI.zip
+  HASH:=skip
+endef
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	unzip $(DL_DIR)/AmuleWebUI.zip -d $(PKG_BUILD_DIR)
+	
+endef
+
 CONFIGURE_ARGS+= \
 	--disable-nls \
 	--disable-static \
@@ -78,6 +91,8 @@ define Package/amule/install
 
 	$(INSTALL_DIR) $(1)/usr/share
 	$(CP) $(PKG_INSTALL_DIR)/usr/share/amule $(1)/usr/share/
+	$(CP) $(PKG_BUILD_DIR)/AmuleWebUI-Reloaded-master $(1)/usr/share/amule/webserver/AmuleWebUI-Reloaded
 endef
 
+$(eval $(call Download,AmuleWebUI))
 $(eval $(call BuildPackage,amule))
